tools/packchk/test/data/TestPackDot/TestVendor.TestPackDot.pdsc

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<package schemaVersion="1.4" xmlns:xs="http://www.w3.org/2001/XMLSchema-instance" xs:noNamespaceSchemaLocation="PACK.xsd">
+  <vendor>TestVendor</vendor>
+  <url>http://www.testurl.com/pack/</url>
+  <name>TestPackDot</name>
+  <description>TestPackDot</description>
+
+  <releases>
+    <release version="0.0.1" date="2021-09-06">>
+      Initial release of TestInvalidPack.
+    </release>
+  </releases>
+
+  <keywords>
+    <keyword>TestPackDot</keyword>
+  </keywords>
+
+  <conditions>
+    <condition id="Test_Condition">
+      <description>Test Device</description>
+      <require Dvendor="ARM:82"/>
+    </condition>
+  </conditions>
+
+  <components>
+    <component Cclass="TestClass" Cgroup="TestGlobal" Cversion="1.0.0" condition="Test_Condition">
+      <description>TestGlobal</description>
+      <files>
+        <file category="source" name=".Files/test1.c"/>
```

Comment:
What are we trying to test here? Filenames starting with a `.` should be legal. At least on unix-like systems such files/directories are considered "hidden".
Didn't we discuss (and test) for names starting with `./`? I.e., the unix-like shortcut for "current working directory"?

Reply:
Okay, so this test actually does not verify the removal of `./` prefixes, intentionally. So we definitively need to add some more tests, probably unit tests, to really cover the implementation.